### DISCUSSION TITLE
Insufficient Logs at RustDesk server

### DISCUSSION THREAD

**Discussion by girishpadia** [rustdesk/rustdesk-server#304] [Q&A]
I have installed rustdesk server. My configuration file is as follow.

`[Unit]
Description=Rustdesk Relay Server
[Service]
Type=simple
LimitNOFILE=1000000
ExecStart=/opt/rustdesk/hbbr 
WorkingDirectory=/opt/rustdesk/
User=root
Group=root
Restart=always
StandardOutput=append:/var/log/rustdesk/relayserver.log
StandardError=append:/var/log/rustdesk/relayserver.error
RestartSec=10
[Install]
WantedBy=multi-user.target
`

When clients connect to each other, there are no logs/errors at `/var/log/rustdesk` location. Further, I want to log all the activities happening at client side (like file transfer, file detletion, etc) on server with reference to every client instances. In case of incident response, these logs are useful to trace malicious activities.

**Comment by dinger1986** [comment]
RustDesk Server Pro does this

**Reply by girishpadia** [reply]
Thank you for the answer. Is RustDesk Server Pro free for self hosting?
